jupyter_book/book_template/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM jekyll/jekyll:3.8.5
+COPY Gemfile .
+RUN apk --update add make gcc g++ libc-dev
+RUN gem install bundler
+RUN bundle install
```

Comment:
There is one other potential issue with this. How will I use jupyter-book with mybinder? Because mybinder expects a requirements.txt or Dockerfile.

Reply:
What might be ideal is to move the Dockerfile itself up a directory (or two) and encourage users to pull the image down from Dockerhub. Does Jupyter have a Dockerhub account, @choldgraf ?

Reply:
hmmm, that is a good question...@carreau any chance you know about the docker images and jupyter?

Reply:
Currently hosted on my personal DockerHub account. Happy to move it, though :)